### DISCUSSION TITLE
feat(Repeater): 增加无数据时是否显示 ContainerTemplate

### DISCUSSION THREAD

**Discussion by shur520** [dotnetcore/BootstrapBlazor#3734] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

![image](https://github.com/dotnetcore/BootstrapBlazor/assets/76552883/55a1c241-4c1e-45a1-8f44-68720ad69d9c)

需求是在表格第一行展示录入框,点保存后下面行显示内容

### Describe the solution you'd like

希望Repeater增加一个设置,无数据时是否显示ContainerTemplate内容

**Comment by bb-auto** [comment]
@shur520 Thank you for contacting us. We will give feedback later.

**Reply by ArgoZhang** [reply]
@shur520 我看了一下代码，没有那么简单增加一个参数就可以实现这个逻辑，我想到的折中方案是，如果数据源无数据，你认为给一个特殊的 `Item` 这样就都可以走通，你在 `ItemTemplate` 模板中做一个简单的判断如果是你放置的特殊 `Item` 就走你那个特定逻辑即可解决
